packages/adblocker-webextension-example/background.ts

```diff
@@ -58,6 +57,7 @@ chrome.tabs.onUpdated.addListener((tabId, { status, url }) => {
 declare global {
   interface Window {
     adblocker: WebExtensionBlocker;
+    browser: Browser;
```

Comment:
From usage below it seems like this is not always defined. Should the type reflect that?
```suggestion
    browser?: Browser;
```

Reply:
My carelessness on that point. Thanks.